src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/config/WebClientConfiguration.kt

```diff
@@ -46,91 +47,55 @@ class WebClientConfiguration(
 
   @Bean
   fun oauthPrisonerSearchClient(authorizedClientManager: OAuth2AuthorizedClientManager): WebClient {
-    val oauth2Client = ServletOAuth2AuthorizedClientExchangeFilterFunction(authorizedClientManager)
-    oauth2Client.setDefaultClientRegistrationId(HMPPS_AUTH)
+    val oauth2Client = getOAuthClient(authorizedClientManager)
```

Comment:
This is a nice improvement
There's also [some stuff](https://github.com/ministryofjustice/hmpps-kotlin-lib/blob/a861941fb3eaf95011c764757834430e5f386d11/test-app/src/main/kotlin/uk/gov/justice/digital/hmpps/testapp/config/WebClientConfiguration.kt#L8) built in the kotlin lib we could possibly consider using at some point